core/src/validation.jl

```diff
@@ -441,6 +441,39 @@ function valid_demand(
     return !errors
 end
 
+function valid_outlet_crest_level!(graph::MetaGraph, outlet::Outlet, basin::Basin)::Bool
+    errors = false
+    for (index, (id, crest)) in enumerate(zip(outlet.node_id, outlet.min_crest_level))
+        id_in = inflow_id(graph, id)
+        if id_in.type == NodeType.Basin
+            #validation:
+            basin_bottom_level = basin_bottom(basin, id_in)
+            if crest == -Inf
+                outlet.min_crest_level[index] = basin_bottom_level[2]
+            elseif crest < basin_bottom_level[2]
+                @error "Minimum crest level of $id is lower than bottom of upstream $id_in"
```

Comment:
```suggestion
                @error "Minimum crest level of $id is lower than bottom of upstream $id_in" crest basin_bottom_level
```

This shows the values directly, also good to do below. It will show the variable name as well as the value, so the variable name should be clear, otherwise you can use `@error "asda" clear_name = a[idx][2]` to rename.